### PR TITLE
Log auxillary function invocations

### PR DESCRIPTION
**Pull request by peterbarker**

### CODE REVIEW

**Comment by peterbarker** [comment]
```
pbarker@bluebottle:~/rc/ardupilot(pr/log-auxfunc)$ mavlogdump.py logs/00000002.BIN --t AUXF
2021-04-28 19:11:05.49: AUXF {TimeUS : 60518283, function : 70, pos : 2, source : 4, result : 1}
pbarker@bluebottle:~/rc/ardupilot(pr/log-auxfunc)$ 
```

File: libraries/RC_Channel/RC_Channel.cpp [hunk 1]
```diff
@@ -893,6 +893,31 @@ void RC_Channel::do_aux_function_mission_reset(const AuxSwitchPos ch_flag)
     mission->reset();
 }
 
+bool RC_Channel::run_aux_function(aux_func_t ch_option, AuxSwitchPos pos, AuxFuncTriggerSource source) {
```
Comment (tridge) [inline libraries/RC_Channel/RC_Channel.cpp:896]: { on next line

**Review by tridge — APPROVED** [review]

File: libraries/RC_Channel/RC_Channel.cpp (hunk 1 above)
Comment (peterbarker) [reply libraries/RC_Channel/RC_Channel.cpp]: Congratulation!  You're the proud winner of 127 `const` review comments :-P

Change made.

File: ArduCopter/RC_Channel.cpp [hunk 2]
```diff
@@ -113,21 +113,19 @@ void RC_Channel_Copter::init_aux_function(const aux_func_t ch_option, const AuxS
...
+        run_aux_function(ch_option, ch_flag, AuxFuncTriggerSource::INIT);
         break;
     default:
         RC_Channel::init_aux_function(ch_option, ch_flag);
         break;
     }
 }
 
-// do_aux_function_change_mode - change mode based on an aux switch
+// run_aux_function_change_mode - change mode based on an aux switch
```
Comment (khancyr) [inline ArduCopter/RC_Channel.cpp:125]: wrong change

**Review by khancyr — COMMENTED** [review]
miss blimp and rover

**Comment by khancyr** [comment]
RC_Channel::init_aux_function should be modify too

File: ArduCopter/RC_Channel.cpp (hunk 2 above)
Comment (peterbarker) [reply ArduCopter/RC_Channel.cpp]: Fixed, thanks.

**Comment by peterbarker** [comment]
> RC_Channel::init_aux_function and RC_Channel::read_aux() should be modify too

How so?

**Comment by peterbarker** [comment]
@khancyr feel free to fill me in on those changes with a follow-up PR :-)

**Pull request merged**